config.example.py

```diff
@@ -31,7 +33,9 @@
       'db_review': 'db/*',
       'css_review': 'app/assets/stylesheets/global/*',
       'js_review': 'node_modules/*',
-      'ops_review': ['config/nginx/*', 'config/settings/*']
+      'ops_review': ['config/nginx/*', 'config/settings/*'],
+      # Only matches files in this folder, not any subfolder
+      'test_review': re.compile(r'test\/[a-z_]+.rb'),
```

Comment:
Guess I'm not sure if this config example is what we actually plan to use in practice. Just be aware of this when you update the actual config.